IDEAS/Examples/IBPSA/SingleZoneResidentialHydronicHeatPump.mo

```diff
@@ -0,0 +1,709 @@
+<td><p>0.009</p></td>
+<td><p>0.14</p></td>
+<td><p>900</p></td>
+<td><p>530</p></td>
+</tr>
+<tr>
+<td><p>Layer 2 (fiber glass)</p></td>
+<td><p>0.1118</p></td>
+<td><p>0.04</p></td>
+<td><p>840</p></td>
+<td><p>12</p></td>
+</tr>
+<tr>
+<td><p>Layer 3 (plaster board)</p></td>
+<td><p>0.01</p></td>
+<td><p>0.16</p></td>
+<td><p>840</p></td>
+<td><p>950</p></td>
+</tr>
+</table>
+<h4>Occupancy schedules</h4>
+<p>The zone is occupied by 5 people before 7 am and after 8 pm each weekday and 
+full time during weekends. </p>
+<h4>Internal loads and schedules</h4>
+<p>There are no internal loads other than the occupants. </p>
+
+<h4>Climate data</h4>
+<p>The model uses a climate file containing one year of weather data for Uccle, 
+Belgium. </p>
+
+<h3>HVAC System Design</h3>
+<h4>Primary and secondary system designs</h4>
+<p>An air-to-water modulating heat pump of 15 kW nominal heating capacity 
+substracts energy from the ambient air to heat up the floor heating emission 
+circuit. A fan blows ambient air through the heat pump evaporator 
+when the heat pump is working to substract heat from the ambient.
+The floor heating presents 
+heat injection between Layer 2 (insulation) and Layer 3 (screed), with water as 
+working fluid which is moved at a flow rate of 0.5 kg/s when the heat pump 
+is working. </p>
+
+<h4>Equipment specifications and performance maps</h4>
+<p><b>Heat pump</b> </p>
+<p>A water-to-air heat pump with a scroll compressor is used. 
+The heat pump is modeled as described by: </p>
+<p>H. Jin. <i>Parameter estimation based models of water source heat pumps. 
+</i>PhD Thesis. Oklahoma State University. Stillwater, Oklahoma, USA. 2012. </p>
+<p>with air instead of water blowing through the evaporator. Air condensation is
+therefore neglected. The model parameters are obtained by calibration of the heat pump model to manufacturer 
+performance data following the procedure explained in
+<a href=\"https://simulationresearch.lbl.gov/modelica/releases/latest/help/Buildings_Fluid_HeatPumps_Calibration.html\">
+this heat pump calibration guide</a>
+using manufacturer performance data from a Carrier air-to-water heat pump model 30AW015. 
+</p>
+<p>Variable speed is achieved by multiplying the full load suction volume flow rate by the 
+normalized compressor speed. The power and heat transfer rates are forced to zero if the 
+resulting heat pump state has higher evaporating pressure than condensing pressure. </p>
+
+<p>Parameters <span style=\"font-family: Courier New;\">TConMax</span> and 
+<span style=\"font-family: Courier New;\">TEvaMin</span> are used to set an upper and lower bound for the condenser 
+and evaporator. The compressor is disabled when these conditions are not satisfied, or when the evaporator 
+temperature is larger than the condenser temperature. This mimics the temperature protection of the heat pump. </p>
+<p>The compression process is assumed isentropic. The thermal energy of superheating is ignored in the evaluation 
```

Comment:
are you sure? the heat pump record has parameter `dTSup`, which seems to be related to super heating.

Reply:
I got that one from the `ScrollWaterToWater` documentation. I think that the `dTSup` parameter is only used by the heat pump to calculate the suction point of the compressor in here: https://github.com/open-ideas/IDEAS/blob/1eb6bbf9cb6f86b93826ed48aeccd3b29c1866f7/IDEAS/Fluid/HeatPumps/Compressors/ScrollCompressor.mo#L78 
which affects the power energy use, but it's not used anywhere else of the heat pump model meaning that thermal energy of superheating is indeed neglected in the evaporator.

Reply:
fair enough